components/button/themes/express.css

```diff
@@ -37,5 +37,12 @@
 			--spectrum-button-background-color-down: var(--spectrum-neutral-background-color-down);
 			--spectrum-button-background-color-focus: var(--spectrum-neutral-background-color-key-focus);
 		}
+
```

Comment:
@TarunAdobe I should have been clearer in the last feedback I left, sorry! This `themes/express.css` file imports `themes/spectrum.css` so the styles from `spectrum.css` are inherited by default here, I don't think anything needs to be added to this theme for this issue.